app/services/hyrax/workflow/workflow_importer.rb

```diff
@@ -60,25 +74,34 @@ def validate!
 
       public
 
+      attr_accessor :errors
+
       def call
+        self.errors = []
         Array.wrap(data.fetch(:workflows)).map do |configuration|
-          find_or_create_from(configuration: configuration)
+          begin
+            find_or_create_from(configuration: configuration)
+          rescue InvalidStateRemovalException => e
+            error = I18n.t('hyrax.workflow.load.state_error', workflow_name: e.state.workflow.name, state_name: e.state.name, entity_count: e.state.entities.count)
+            Rails.logger.error(error)
+            errors << error
+            Sipity::Workflow.find_by(name: configuration[:name])
+          end
         end
       end
 
       private
 
         def find_or_create_from(configuration:)
-          workflow = Sipity::Workflow.find_or_initialize_by(name: configuration.fetch(:name)) do |wf|
```

Comment:
@cam156 I'll write up a spec (but for now consider this one tabled and okay to keep as you have).